ls1intum/Artemis
# `Lectures`: Support import of online units during lecture import

### Checklist
#### General


#### Server

### Motivation and Context



Online units were not imported during lecture import but they should be.

### Description


Added it.

### Steps for Testing


1. Create a lecture with an online unit
2. Import the lecture to somewhere else
3. Check that the imported lecture has an online unit

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
